### ISSUE TITLE
Question about advantage calculation

### ISSUE THREAD

**Issue by leeacord** [danijar/dreamerv2#60]
Hi,

I have a question regarding the implementation of the advantage calculation. The code snippet is as follows:
https://github.com/danijar/dreamerv2/blob/07d906e9c4322c6fc2cd6ed23e247ccd6b7c8c41/dreamerv2/agent.py#L252-L274

```python
advantage = tf.stop_gradient(target[1:] - self._target_critic(seq['feat'][:-2]).mode())
```

Based on my understanding:

- `seq['feat']` contains time steps from `0` to `horizon`.
- `target` contains time steps from `0` to `horizon-1`, since the value at the last step is used as a bootstrap for `lambda_return`.
- Therefore, `baseline` in Line 271 includes time steps from `0` to `horizon-2`, and `target[1:]` includes time steps from `1` to `horizon-1`.

If I understand correctly, the code uses $V_{t+1}^{\lambda} - v_\xi\left(\hat{z}_t\right)$ to calculate the advantage, 

not $V_t^{\lambda} - v_{\xi}(\hat{z}_t)$  as stated in the paper?

I am very impressed with the Dreamer series of reinforcement learning algorithms. Thank you for your hard work!
